devbuild.sh

```diff
@@ -0,0 +1,19 @@
+VERSION="$(git describe --tags --always --abbrev=0 --match='v[0-9]*.[0-9]*.[0-9]*' 2> /dev/null | sed 's/^.//')"
+
+ARCH="$(uname -m | tr '[:upper:]' '[:lower:]')"
+
+if [ "$ARCH" = "x86_64" ]; then
+  ARCH="amd64"
+fi
+
+LDFLAGS=(
+  "-X 'encr.dev/internal/version.Version=${VERSION}'"
+  "-X 'main.defaultGitRemoteName=local'"
+  "-X 'main.defaultGitRemoteURL=ssh://localhost:9040/'"
+  "-X 'encr.dev/internal/conf.defaultPlatformURL=http://localhost:9000'"
```

Comment:
These three only make sense if you are able to run our platform locally; I think for this script we should remove them...